api/integrations/hg/zulip-changegroup.py

```diff
@@ -101,13 +105,15 @@ def send_zulip(email, api_key, site, stream, subject, content):
     client.send_message(message_data)
 
 def get_config(ui, item):
+    # type: (Any, str) -> Optional[str]
     try:
         # configlist returns everything in lists.
         return ui.configlist('zulip', item)[0]
     except IndexError:
         return None
 
 def hook(ui, repo, **kwargs):
+    # type: (Any, Any, Optional[Text]) -> None
```

Comment:
You can check the types of `ui` and `repo` in [Mercurial API](https://www.mercurial-scm.org/wiki/MercurialApi).